key-value/go/Makefile

```
@@ -1,11 +1,11 @@
-.PHONY: all clean
+ifndef GO
+GO := go
+endif
 
-GO = $(LANGBENCH)/runtimes/go/go-1.14.1-vanilla/go
+compile:
+	$(GO) build -o key_value.o
```

```suggestion
	$(GO) build -o key_value.o key_value.go hashmap.go
```

I think you need the filenames to be able to compile without creating a module etc.